lambdas/search/index.py

```diff
@@ -46,8 +44,10 @@ def lambda_handler(request):
     """
     Proxy the request to the elastic search.
     """
+
     action = request.args.get('action')
     indexes = request.args.get('index')
+    terminate_after = os.getenv('MAX_DOCUMENTS_PER_SHARD')
```

Comment:
Is `None` really a safe default? We should probably default to an integer?

Also, you need to convert from string (`os.getenv()`) to int (what the API expects). `int("1,000")` will fail because of commas:
```
locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
locale.atoi(terminate_after)
```
Docs are not really clear on kwarg type https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch.search

Reply:
None is a safe default. The argument is dropped before sending. See the example url in test_search.py (test_stats).

I didn't convert the string because it isn't necessary since it's being encoded and sent as a string. I also wanted to preserve None (though I could do that manually).

Reply:
OK but what if the user sets a string that is not an integer?

Reply:
It's set by the stack (or lambda deployment).